### ISSUE TITLE
Bug when receiving a command with “x é” 

### ISSUE THREAD

**Issue by irevoire** [serenity-rs/serenity#823]
Hello,
I found a strange bug, when I execute any command with something finishing by ` é` serenity panic.
For example:
```
!bug x é # panic
!bug aa é # panic
!bug bonjour éol # panic
!bug ééé # do not panic
!bug x è # panic
!bug x ê # panic
```

Here is a minimal repo where I can reproduce the bug: https://github.com/irevoire/bug

**Comment by irevoire** [comment]
How much time should we wait till this is uploaded to crates.io ?

**Comment by arqunis** [comment]
Not long. I have just released 0.8.3 that includes my patch, and so the issue should be gone.
